From b6428e00842db7a3d7eaa259adc34950552b661a Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Tue, 29 Sep 2026 11:21:08 +0000
Subject: [PATCH] Refresh the guest package lock

appstream 1.2.0-1 -> 1.2.1-1
automake 1.18.1-1 -> 1.19-1
coreutils 9.11-2 -> 9.12-2
fastfetch 2.68.1-1 -> 2.69.0-1
glibc 2.44+r24+g16be1518495f-1 -> 2.44+r50+g1848099f063e-1
imagemagick 7.1.2.31-2 -> 7.1.2.32-1
inotify-tools 4.25.9.0-1 -> 4.26.262-1
libvips 8.18.6-2 -> 8.18.7-1
mkinitcpio 42-1 -> 42.1-1
python-dbus 1.4.0-2 -> 1.5.0-1
---
 guest/packages.lock.json | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 074fa5c..151979a 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -12,14 +12,14 @@
     "alsa-topology-conf": "1.2.5.1-4",
     "alsa-ucm-conf": "1.2.16.1-1",
     "aom": "3.15.1-1",
-    "appstream": "1.2.0-1",
+    "appstream": "1.2.1-1",
     "aquamarine": "0.15.1-1",
     "archlinux-keyring": "20260909-1",
     "at-spi2-core": "2.60.7-1",
     "attr": "2.6.0-1",
     "audit": "4.2.1-1",
     "autoconf": "2.73-1",
-    "automake": "1.18.1-1",
+    "automake": "1.19-1",
     "avahi": "1:0.9rc5-1",
     "base": "3-3",
     "base-devel": "1-2",
@@ -42,7 +42,7 @@
     "chromium": "153.0.8010.52-1",
     "clang": "22.1.8-1",
     "compiler-rt": "22.1.8-1",
-    "coreutils": "9.11-2",
+    "coreutils": "9.12-2",
     "cpio": "2.15-3",
     "cpptrace": "1.0.4-2",
     "cryptsetup": "2.8.8-1",
@@ -71,7 +71,7 @@
     "expat": "2.8.5-1",
     "eza": "0.23.5-2",
     "fakeroot": "1:1.37.2-3",
-    "fastfetch": "2.68.1-1",
+    "fastfetch": "2.69.0-1",
     "fcft": "3.3.3-1",
     "fcitx5": "5.1.23-1",
     "fcitx5-gtk": "5.1.7-1",
@@ -108,7 +108,7 @@
     "git": "2.55.0-1",
     "glib-networking": "1:2.80.1-1",
     "glib2": "2.88.3-1",
-    "glibc": "2.44+r24+g16be1518495f-1",
+    "glibc": "2.44+r50+g1848099f063e-1",
     "glslang": "1:1.4.357.0-1",
     "glu": "9.0.3-3",
     "glycin": "2.1.5-2",
@@ -173,11 +173,11 @@
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
-    "imagemagick": "7.1.2.31-2",
+    "imagemagick": "7.1.2.32-1",
     "imath": "3.2.3-1",
     "imv": "5.0.1-2",
     "iniparser": "4.3.0-1",
-    "inotify-tools": "4.25.9.0-1",
+    "inotify-tools": "4.26.262-1",
     "iproute2": "7.2.0-1",
     "iptables": "1:1.8.13-1",
     "iputils": "20250605-1",
@@ -375,7 +375,7 @@
     "libva": "2.24.1-1",
     "libvdpau": "1.5-4",
     "libverto": "0.3.2-6",
-    "libvips": "8.18.6-2",
+    "libvips": "8.18.7-1",
     "libvorbis": "1.3.7-4",
     "libvpl": "2.17.0-1",
     "libvpx": "1.17.0-1",
@@ -440,7 +440,7 @@
     "mdadm": "4.6-2",
     "mesa": "1:26.2.3-1",
     "minizip": "1:1.3.2-3",
-    "mkinitcpio": "42-1",
+    "mkinitcpio": "42.1-1",
     "mkinitcpio-busybox": "1.36.1-1",
     "mobile-broadband-provider-info": "20251101-1",
     "mpdecimal": "4.0.1-3",
@@ -520,7 +520,7 @@
     "psmisc": "23.7-2",
     "pugixml": "1.16-1",
     "python": "3.14.7-1",
-    "python-dbus": "1.4.0-2",
+    "python-dbus": "1.5.0-1",
     "python-docopt": "0.6.2-15",
     "python-gobject": "3.56.3-1",
     "python-pyxdg": "0.28-7",
-- 
2.55.0

